src/ray/common/ray_config_def.h

```diff
@@ -692,7 +692,7 @@ RAY_CONFIG(std::string, predefined_unit_instance_resources, "GPU")
 /// The scheduler will treat these custom resource types as unit_instance.
 /// Default custom_unit_instance_resources is empty.
 /// When set it to "FPGA", we will treat FPGA as unit_instance.
-RAY_CONFIG(std::string, custom_unit_instance_resources, "")
+RAY_CONFIG(std::string, custom_unit_instance_resources, "num_neuron_cores")
```

Comment:
Let's say, any users want to overwrite this RAY_ custom_unit_instance_resources="" for their usage. If anyone does it, the num_neuron_cores will not become custom unit instance resources anymore.

Reply:
Looks like we allow to have a comma separator. So let's imagine someone puts RAY_ custom_unit_instance_resources="FPGA". In this case, we should just append ",num_neuron_cores" instead of allowing the user to overwrite the config?

Reply:
In this case, we should just append ",num_neuron_cores" instead of allowing the user to overwrite the config?
> Yes, that's right since the config supports with comma_separated. And we protect the overwrites with unit-tests.